docs/search/search_for_content.md

```diff
@@ -50,4 +50,4 @@ Depending on the system configuration, a content item count can be indicated for
 Select your filters, then click **Apply** to update the results.
 Click **Clear** to reset and remove all filters.
 
-![Filtered Search](img/filtered_search.png)
+<!--ARCADE EMBED START--><div style="position: relative; padding-bottom: calc(51.27314814814815% + 41px); height: 0; width: 100%;"><iframe src="https://demo.arcade.software/WuOs9pESmyZLhyyqLqfn?embed&embed_mobile=tab&embed_desktop=inline&show_copy_link=true" title="Search for content" frameborder="0" loading="lazy" webkitallowfullscreen mozallowfullscreen allowfullscreen allow="clipboard-write" style="position: absolute; top: 0; left: 0; width: 100%; height: 100%; color-scheme: light;" ></iframe></div><!--ARCADE EMBED END-->
```

Comment:
"Click on the suggestion from the drop-down to see content item..." > "Click on a suggestion from the drop-down to see the content item..."
"Change sorting option" > "Change the sorting order"
"You can use filters to narrow down results list" > "Use filters to narrow down the results list"
"If no results found, you can clear filters and try again" > "If no results are found, clear filters and try again"